app/code/Magento/Ui/view/base/web/js/core/renderer/layout.js

```diff
@@ -600,6 +602,25 @@ define([
             });
 
             return result;
+        },
+
+        /**
+         * Get sort order for child if it is undefined.
+         *
+         * @param {Object} parent
+         * @param {Object} child
+         */
+        getChildSortOrderIfUndefined: function (parent, child) {
+            var key, i = 0;
+
+            for (key in parent.children) {
+                if (child.index === key
+                    && typeof parent.children[key].sortOrder === "undefined") {
```

Comment:
Looks like this component depends on `underscore` already, so it is possible to use it's method to detect undefined.

Reply:
Fixed.